app/.babelrc

```diff
@@ -1,5 +1,13 @@
 {
-  "presets": ["airbnb"],
+  "presets": [
```

Comment:
What does this do? I wouldn't think something in the app folder would affect the jupyter notebook stuff.